filebeat/input/file/state.go

```diff
@@ -8,41 +8,41 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-// FileState is used to communicate the reading state of a file
-type FileState struct {
+// State is used to communicate the reading state of a file
+type State struct {
```

Comment:
BTW: This interface should be provided / used by registrar then to persist any state. So it is available to all beats.